src/App.js

```diff
@@ -1,18 +1,15 @@
-import React from "react";
-import logo from "./logo.svg";
-import { Menu, Grid, Image } from "semantic-ui-react";
-import "./App.css";
+import React from 'react';
+import { Menu, Grid, Image } from 'semantic-ui-react';
+import logo from './logo.svg';
+import './App.css';
 
 function App() {
   return (
     <div>
       <header>
         <Menu stackable>
-          <Menu.Item style={{ flexGrow: "1" }}>
-            <img
-              src="https://react.semantic-ui.com/logo.png"
-              alt="placeholder"
-            />
+          <Menu.Item style={{ flexGrow: '1' }}>
+            <img src="https://react.semantic-ui.com/logo.png" alt="placeholder" />
```

Comment:
These  changes and the single quotes, where made by the linter?

Reply:
Yes, according to the Airbnb code style guide, single quotes are for attributes and double are for other JS. 

https://github.com/airbnb/javascript/tree/master/react#quotes

Reply:
Also the alignment, I guess?

Reply:
Yes, all the changes regarding formatting for App.js and Index.js were done automatically by the linter.